platforms/pf_test/trace_utils.c

```diff
@@ -0,0 +1,62 @@
+/* System includes */
+#include <stdlib.h>
+
+/* RIOT includes */
+#include "xtimer.h"
+
+/* Application includes */
+#include "trace_utils.h"
+#include "tracefd.h"
+#include "platform.h"
+#include "obstacles.h"
+
+/* Trace filename */
+#define TRACE_FILE "trace.txt"
+
+/* Trace file descriptor on sdcard */
+static tracefd_t tracefd_sdcard;
+
+/* Periodic task */
+#define TASK_PERIOD_MS 60
```

Comment:
why 60ms here and 100ms before ?

Reply:
Because it allows the simulator to be more precise and it has few impacts on performance.